writers/UnreliableLocalWriter.go

```diff
@@ -16,6 +15,7 @@ type Error struct {
 	Cause error
 }
 
+// UnreliableWriter Not thread safe, so all methods have to be called from same thread
```

Comment:
It is not required to call all methods from the same thread. It is required that users of an UnreliableWriter synchronise themselves.